include/ql/com/topology.h

```diff
@@ -269,6 +269,11 @@ class Topology {
      */
     utils::Bool is_comm_qubit(Qubit qubit) const;
 
+    /**
+     * Returns whether the two given qubits are communication qubits.
+     */
+    utils::Bool are_comm_qubits(Qubit source, Qubit target) const;
```

Comment:
Is this function really necessary? I think the && with two is_comm_qubits is fine.